### PR TITLE
SRP code executable

### PR DESCRIPTION
**Pull request by artemasmith**

### CODE REVIEW

File: solid/single_responsibility.rb [hunk 1]
```diff
@@ -20,73 +20,127 @@
...
+    private
+
+    # One of the most conspicuous indicators of missing Object is an attribute,
+    # we're passing to many methods
+    def mark_deal_processed(deal)
+      deal.mark_as_processed
+    end
+
+    def calculate_commission(deal)
+      @_calculate_commission ||= deal.amount * 0.05
```
Comment (khusnetdinov) [inline solid/single_responsibility.rb]: remove `_@` variable, use other names

File: solid/single_responsibility.rb [hunk 2]
```diff
@@ -20,73 +20,127 @@
...
+      deal.amount -= calculate_commission(deal)
+    end
+  end
 end
 
 # Solution
 
-class DealProcessor
-  attr_reader :deals
+module SRP
```
Comment (khusnetdinov) [inline solid/single_responsibility.rb]: What is SRP? Use please readable names

File: solid/single_responsibility.rb (hunk 1 above)
Comment (artemasmith) [reply solid/single_responsibility.rb]: removed

File: solid/single_responsibility.rb (hunk 2 above)
Comment (artemasmith) [reply solid/single_responsibility.rb]: it's an acronym of Single Responsibility Principle. I changed namespaces for you wish.

**Pull request closed without merging**